### ISSUE TITLE
As an end-user I would like to see the mock-up / wireframes of the new front-end application based on OTP intermodal routing engine

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/odh-mentor-otp#37]
- [x] first proposal by Open Move 
- [x] joint review during meeting 19.3

Next steps: review and finalization by Open Move:
- [x] review (e.g advanced settings)
- [x] colours (-> current mobility.meran.eu)
- [x] responsive (mobile view)

Final step
- [x] Merano mock-up consolidation (see open points list below)
- [ ] final review and approval by NOI (with Merano stakeholders)

**Comment by rcavaliere** [comment]
Comments agreed:
- [ ] colored bulletin on the icon -> to be substituted with the number of available vehicles, such as on https://analytics.opendatahub.bz.it/
- [ ] gray color to be used for "offline" stations (e.g. in maintenance)
- [ ] primary color: suedtirolmobil blue
- [ ] secondary color to be added for call to actions
- [ ] different tooltip based on data type (e.g. online vs offline parking)

@rcavaliere will also collect additional internal feedback.

Basically the UI concept is agreed, all these details are going to be fine-tuned during the implementation

**Comment by rcavaliere** [comment]
We have reviewed internally and with some interested stakeholder the first mock-ups. In general looks very good and the comments are positive. Main comments are the following:

- language selection: use labels IT, DE, EN (now a standard in the web) and avoid the use of flags
- colours: the Tourist Agency of Merano, which is a historic partner of the beta application mobility.meran.eu (they own the domain meran.eu) suggest to use a different palette of colors, which are the new ones for the city of Merano (see attachment)

I suggest to implement the application so that these colors can be simply changed and configured in the source code
[190218_Merano_Farben.pdf](https://github.com/noi-techpark/odh-mentor-otp/files/6299618/190218_Merano_Farben.pdf)

**Comment by zabuTNT** [comment]
Updated mockups with colors and enhancements

[Wireframe Merano (1).pdf](https://github.com/noi-techpark/odh-mentor-otp/files/6404303/Wireframe.Merano.1.pdf)
[Wireframe Mobile Merano (1).pdf](https://github.com/noi-techpark/odh-mentor-otp/files/6404304/Wireframe.Mobile.Merano.1.pdf)

**Comment by rcavaliere** [comment]
Looks good, I make an immediate question: why didn't you use the colours of Merano (see above)?

**Comment by zabuTNT** [comment]
@rcavaliere 
I'll ask this thing, but the idea it's that anyways all these colors will be easily settable on deployment.

**Comment by rcavaliere** [comment]
OK, that's fine. Would be nice to have also a version of the wireframes also with these colours to be presented with the Merano's stakeholders

**Comment by rcavaliere** [comment]
We have decided to have also a version of the mock-up in the Merano colours.
@rcavaliere will ask the Tourist Agency of Merano how they would suggest to do this, and then we will make a version of this mock-up also according to this suggestion. This will be also the graphical line to be used for our public POC.

**Comment by rcavaliere** [comment]
I have received some inputs from the Tourist Agency of Merano.
The proposal is the following:
- [ ] base colour: green (Merano colour "Salbeigruen"). To be used in the three different gradations for the different elements
- [ ] M blue icon for public transport (the color which is currently used, to be used also for the on-demand services, when available)
- [ ] "ocker" colour for sharing services (i.e. car and bike sharing, maybe with two different graduations)
- [ ] "pastellblau" color for parking
- [ ] "gelb" color for e-mobility charging stations
- [ ] "rose" color for car pooling

Can you please prepare a version of the mock-up with these inputs, so that we can see how it will look like?

**Comment by zabuTNT** [comment]
thank you @rcavaliere, and about the fonts?

**Comment by rcavaliere** [comment]
Final adjustments decided:
- [ ] clustering icon without "sticker" with real-time availability info
- [ ] "sticker" with real-time availability info coloured (green = available; red = not available)
- [ ] texts and numbers in tooltips in black colour
- [ ] transport modes with the above Merano colours + if necessary an alternative proposal of colours for improving the "look & feel"

**Comment by rcavaliere** [comment]
Next step: integrate on-demand + car pooling

**Comment by zabuTNT** [comment]
New Palette
[Desktop.pdf](https://github.com/noi-techpark/odh-mentor-otp/files/6639110/Desktop.pdf)
[Mobile.pdf](https://github.com/noi-techpark/odh-mentor-otp/files/6639111/Mobile.pdf)

**Comment by rcavaliere** [comment]
The new mock-ups are fine. I suggest to close this issue and to other another one for the addition of the layers "car pooling" and "on-demand"
